apps/pie-docs/src/components/chip/overview/overview.md

```diff
@@ -0,0 +1,326 @@
+        ]
+    }
+} %}
+
+___
+
+## Anatomy
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/anatomy.svg",
+    alt: "Anatomy of the chip component.",
+    width: 210
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Icon (Optional):** Non-interactive icon that can be used to visually support the string.",
+        "**String:** Text label informing the user of the option/selection.",
+        "**Close (optional)**: Allows the Chip to be dismissible, but the application should provide a way for them to easily add it back."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-default.svg",
+    alt: "The default variant of the chip component.",
+    width: 63,
+    variant: "secondary"
+} %}
+
+### Outline
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-outline.svg",
+    alt: "The outline variant of the chip component.",
+    width: 63
+} %}
+
+### Ghost
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-ghost.svg",
+    alt: "The ghost variant of the chip component.",
+    width: 63
+} %}
+
+---
+
+## Modifiers
+
+### Icon
+
+Icons are always positioned to the left of the string.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/modifier-icon.svg",
+    alt: "A chip component with a placeholder icon on the left.",
+    width: 87
+} %}
+
+### Tick icon
+
+Tick icon can be added for clarity and decision support when using chips to filter or multi-select to the selected state. 
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/modifier-tick-icon.svg",
+    alt: "A chip component with a tick icon on the left.",
+    width: 63
+} %}
+
+### Close icon
+
+Close icon can be added to the selected state to allow users to remove the selection or filter.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/modifier-close-icon.svg",
+    alt: "A chip component with a close icon on the right.",
+    width: 63
+} %}
+
+---
+
+## Size
+
+Chips have a minimum width of 48px.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/size.svg",
+    alt: "A chip component with the value of 1.",
+    width: 48
+} %}
+
+---
+
+## Content
+
+- Keep the strings short so they are easy to read and scan.
+- Use sentence case.
+
+---
+
+## Layout
+
+If there is a Chip grouping, by default are laid out horizontally and stack if required. A spacing of 8px is used horizontally, and 12px for vertical stacking.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/chip/layout-single-row.svg",
+      width: 237,
+      alt: "A group of chip components laid out horizontally",
+      variant: "secondary"
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/chip/layout-double-row.svg",
```

Comment:
Personally I think this screenshot would be more helpful if the chips contained words of different lengths so that you can see it's not just a grid.

Incidentally, is there any guidance about what words or phrases you should put inside a chip? Single words only, perhaps, or are multiple words in sentence-case allowed?

Reply:
@tingting-lau can you please advice here? ⭐

Reply:
images updated, please check the google sheet